pilot/pkg/proxy/envoy/v1/discovery_test.go

```diff
@@ -328,6 +328,36 @@ func TestClusterDiscoveryWithSecurityOn(t *testing.T) {
 	compareResponse(response, "testdata/cds-ssl-context.json", t)
 }
 
+func TestClusterDiscoveryWithSecurityOnByByAuthenticationPolicy(t *testing.T) {
```

Comment:
Since we have some older tests doing the same, can you remove those?

Reply:
I'd like to keep both until we deprecate the flag/annotation way to enable mTLS.